bin/tools/frontend-tests/tpl/service_as_class.twig

```diff
@@ -0,0 +1,32 @@
+// THIS TEST WAS GENERATED AUTOMATICALLY ON {{ createdAt }}
```

Comment:
You can use `{{ "now"|date("m/d/Y H:i:s") }}` filter to get current time. 
See: http://twig.sensiolabs.org/doc/filters/date.html